.github/workflows/checks.yml

```diff
@@ -19,67 +19,59 @@ jobs:
   markdownlint:
     runs-on: ubuntu-latest
     steps:
-    - name: check out code
-      uses: actions/checkout@v4
-
-    - name: install dependencies
-      run: npm install
-
-    - name: run markdownlint
-      run: make markdownlint
+      - name: check out code
+        uses: actions/checkout@v4
+      - name: install dependencies
+        run: npm install
+      - name: run markdownlint
+        run: make markdownlint
```

Comment:
Shouldn't we get rid of this one?

Reply:
No, this would be for markdownlint (not markdownlinkcheck)